config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-2.yaml

```diff
@@ -250,7 +250,7 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.2
       testgrid-tab-name: capi-pr-e2e-full-release-1-2
-  - name: pull-cluster-api-e2e-workload-upgrade-1-26-latest-release-1-2
+  - name: pull-cluster-api-e2e-workload-upgrade-1-27-latest-release-1-2
```

Comment:
I think we should align this job to periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-release-1-2

As v1.26 is the highest version v1.2 will ever support